### PR TITLE
Revert "Revert "[ALLUXIO-1749] Make thrift largest frame size adjustable""

### PR DESCRIPTION
**Pull request by apc999**
Reverts Alluxio/alluxio#2830

We are able to reproduce the problem of complaining frame size smaller than the default limit. As a result, we increase the frame size and add the support of customizing this value.

### CODE REVIEW

**Comment by calvinjia** [comment]
@apc999 Do we want to mark this experimental?

**Comment by apc999** [comment]
@calvinjia experimental marked

File: core/common/src/main/resources/alluxio-default.properties [hunk 1]
```diff
@@ -24,6 +24,7 @@ alluxio.keyvalue.enabled=false
 alluxio.keyvalue.partition.size.bytes.max=512MB
 alluxio.metrics.conf.file=${alluxio.home}/conf/metrics.properties
 alluxio.network.host.resolution.timeout.ms=5000
+alluxio.network.thrift.frame.size.bytes.max=64MB
```
Comment (calvinjia) [inline core/common/src/main/resources/alluxio-default.properties]: Do we want to make the default 16MB so there is no change by default?

File: core/common/src/main/resources/alluxio-default.properties [hunk 2]
```diff
@@ -24,6 +24,7 @@ alluxio.keyvalue.enabled=false
 alluxio.keyvalue.partition.size.bytes.max=512MB
 alluxio.metrics.conf.file=${alluxio.home}/conf/metrics.properties
 alluxio.network.host.resolution.timeout.ms=5000
+alluxio.network.thrift.frame.size.bytes.max=16MB
```
Comment (jsimsa) [inline core/common/src/main/resources/alluxio-default.properties]: should this be 64MB?

Comment (apc999) [inline core/common/src/main/resources/alluxio-default.properties]: fixed.

File: core/common/src/main/resources/alluxio-default.properties (hunk 1 above)
Comment (apc999) [inline core/common/src/main/resources/alluxio-default.properties]: changed to 16 MB

**Comment by apc999** [comment]
@calvinjia @jsimsa  thanks for the suggestion. I fixed your comments

**Comment by jsimsa** [comment]
LGTM

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**